lib/x11utils.pm

```diff
@@ -112,6 +120,8 @@ sub ensure_unlocked_desktop {
                     assert_screen 'generic-desktop';
                 }
                 else {
+                    diag("Next loop ($counter), Generic desktop didn't match");
+                    record_info('Screen seems frozen', 'Might be consequence of bsc#1168979') if is is_aarch64;
```

Comment:
The additional `is` in `if is is_aarch64` looks like a typo, no?

Reply:
Yeah, just fixed it

Reply:
Nice catch!

Reply:
Good that I waited for travis: https://travis-ci.org/github/os-autoinst/os-autoinst-distri-opensuse/jobs/685624978#L16274